### DISCUSSION TITLE
Stop radio stream

### DISCUSSION THREAD

**Discussion by Herrhomebert** [schreibfaul1/ESP32-audioI2S#185] [Q&A]
Hi Schreibfaul,
is there a way to stop the radio playing? Something like audio.disconnect()?

**Comment by biologist79** [comment]
[audio.stopSong()](https://github.com/schreibfaul1/ESP32-audioI2S/blob/master/src/Audio.h#L165)
[audio.pauseResume()](https://github.com/schreibfaul1/ESP32-audioI2S/blob/master/src/Audio.h#L162)
